checks.d/network.py

```diff
@@ -275,36 +294,296 @@ def _check_linux(self, instance):
+                'TCPAbortOnLinger'	        : 'system.net.tcpext.abort_on_linger',
+                'TCPAbortFailed'	        : 'system.net.tcpext.abort_failed',
+                'TCPMemoryPressures'	    : 'system.net.tcpext.memory_pressures',
+                'TCPSACKDiscard'	        : 'system.net.tcpext.sack_discard',
+                'TCPDSACKIgnoredOld'	    : 'system.net.tcpext.dsack_ignore_old',
+                'TCPDSACKIgnoredNoUndo'	    : 'system.net.tcpext.dsack_ignore_no_undo',
+                'TCPSpuriousRTOs'	        : 'system.net.tcpext.spurious_rtos',
+                'TCPMD5NotFound'	        : 'system.net.tcpext.md5_not_found',
+                'TCPMD5Unexpected'	        : 'system.net.tcpext.md5_unexpected',
+                'TCPSackShifted'	        : 'system.net.tcpext.sack_shifted',
+                'TCPSackMerged'	            : 'system.net.tcpext.sack_merged',
+                'TCPSackShiftFallback'	    : 'system.net.tcpext.sack_shift_fallback',
+                'TCPBacklogDrop'	        : 'system.net.tcpext.backlog_drop',
+                'TCPMinTTLDrop'	            : 'system.net.tcpext.min_ttl_drop',
+                'TCPDeferAcceptDrop'	    : 'system.net.tcpext.defer_accept_drop',
+                'IPReversePathFilter'	    : 'system.net.tcpext.reverse_path_filter',
+                'TCPTimeWaitOverflow'	    : 'system.net.tcpext.time_wait_overflow',
+                'TCPReqQFullDoCookies'	    : 'system.net.tcpext.req_q_full_do_cookies',
+                'TCPReqQFullDrop'	        : 'system.net.tcpext.req_q_full_drop',
+                'TCPRetransFail'	        : 'system.net.tcpext.retrans_fail',
+                'TCPRcvCoalesce'	        : 'system.net.tcpext.rcv_coalesce',
+                'TCPOFOQueue'	            : 'system.net.tcpext.ofo_queue',
+                'TCPOFODrop'	            : 'system.net.tcpext.ofo_drop',
+                'TCPOFOMerge'	            : 'system.net.tcpext.ofo_merge',
+                'TCPChallengeACK'	        : 'system.net.tcpext.challenge_ack',
+                'TCPSYNChallenge'	        : 'system.net.tcpext.syn_challenge',
+                'TCPFastOpenActive'	        : 'system.net.tcpext.fast_open_active',
+                'TCPFastOpenPassive'	    : 'system.net.tcpext.fast_open_passive',
+                'TCPFastOpenPassiveFail'	: 'system.net.tcpext.fast_open_passive_fail',
+                'TCPFastOpenListenOverflow'	: 'system.net.tcpext.fast_open_listen_overflow',
+                'TCPFastOpenCookieReqd'	    : 'system.net.tcpext.fast_open_cookie_reqd',
+                'TCPSpuriousRtxHostQueues'	: 'system.net.tcpext.spurious_rtx_host_queues',
+                'BusyPollRxPackets'	        : 'system.net.tcpext.busy_poll_rx_packets'
+            }
+
+            for key, metric in tcpext_metrics_name.iteritems():
+                if key in tcpext_metrics:
+                    self.rate(metric, self._parse_value(tcpext_metrics[key]))
```

Comment:
Wouldn't a monotonic_count be a more accurate representation of these values? IMO, they should show up as counters within DataDog.